bin/create-app-secret

```diff
@@ -1,18 +1,19 @@
 #!/bin/bash
+set -e
```

Comment:
This effectively breaks `--override-conf-dir` and `-v`.

When both are passed, script will call `diff --suppress-common-line -y` to display the differences between file from repo and one that will be used by quadlet - most likely to aid debugging `override_unit_section` and friends.

But `diff` will return exit code 1 if there are any differences, and `set -e` will stop script execution on first command that does not return 0.

We can put `set +e` / `set -e` around `diff`, or add `|| true` and the end of diff command.

Reply:
Good catch! I added `set -e` mostly as a personal safeguard at the last minute, not because anything really demands it. If adding exceptions and trues becomes too onerous, I might just drop it, but first I'll try adding more trues.

Reply:
I re-read our scripts, and _I think_ these `diff`s were the only remaining calls that could produce nonzero exit codes that we do not want to halt execution. I chose to add `|| true`s to them because I also already added `|| true` to a few other `podman` and `systemctl` commands that may produce acceptable nonzero exit codes.

I squashed the previous fixup commit and amended the additional `|| true`s to the initial commit.